src/main/java/com/cbms/ui/controller/AddSystemTypeController.java

```diff
@@ -0,0 +1,152 @@
+package com.cbms.ui.controller;
+
+import com.cbms.app.item.AssetType;
+import com.cbms.app.item.AssetTypeParameter;
+import com.cbms.source.local.AssetTypeDAOImpl;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.paint.Paint;
+import javafx.scene.shape.StrokeType;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+
+public class AddSystemTypeController implements Initializable {
+
+    @FXML
+    private Button systemMenuButton;
+    @FXML
+    private Button cancelButton;
+    @FXML
+    private Button addThresholdBtn;
+    @FXML
+    private AnchorPane systemTypeInformation;
+    @FXML
+    private Button savebtn;
+    @FXML
+    private TextField systemTypeName;
+
+    private int thresholdCount = 1;
+    private double spacing = 40.0;
+    private UIUtilities uiUtilities;
+    private ArrayList<TextField> thresholdValues;
+    private AssetTypeDAOImpl db;
+    private ArrayList<AssetTypeParameter> assetTypeParameters;
+
+    /**
+     * Initialize runs before the scene is displayed.
+     * It initializes elements and data in the scene.
+     *
+     * @param url
+     * @param resourceBundle
+     *
+     * @author Jeff
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle resourceBundle) {
+        thresholdValues = new ArrayList<>();
+        uiUtilities = new UIUtilities();
+        db = new AssetTypeDAOImpl();
+        assetTypeParameters = new ArrayList<>();
+        assetTypeParameters.add(new AssetTypeParameter());
+        attachEvents();
+    }
+
+    /**
+     * Attaches events to elements in the scene.
+     *
+     * @author Jeff
+     */
+    public void attachEvents() {
+        // Change scenes to Systems.fxml
+        systemMenuButton.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent mouseEvent) {
+                uiUtilities.changeScene(mouseEvent, "/Systems");
+            }
+        });
+        // Change scenes to Systems.fxml
+        cancelButton.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent mouseEvent) {
+                uiUtilities.changeScene(mouseEvent, "/Systems");
+            }
+        });
+        addThresholdBtn.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent mouseEvent) {
+                addThresholdForm();
+            }
+        });
+        savebtn.setOnMouseClicked(new EventHandler<MouseEvent>() {
```

Comment:
Cap on value
My understanding is that the threshold value is a % of the RUL, meaning that there should be a min value (0) and a max value (100). This should be enforced in the controller.